rust/main/Cargo.toml

```diff
@@ -13,7 +13,7 @@ members = [
   "ethers-prometheus",
   "hyperlane-base",
   "hyperlane-core",
-  "hyperlane-test",
+  "hyperlane-test", "submitter",
```

Comment:
```suggestion
  "hyperlane-test", 
  "submitter",
```

Reply:
I think it got auto-populated or formatted like this, let me see